index.html

```diff
@@ -450,13 +450,13 @@ <h3>Abstract Roles</h3>
 		</section>
 		<section id="requiredState">
 			<h3>Required States and Properties</h3>
-			<p><a>States</a> and <a>properties</a> specifically required for the <a>role</a> and subclass roles. Content authors MUST provide a non-empty <span>value</span> for required states and properties. Content authors MUST NOT use the value <code>undefined</code> for required states and properties, unless <code>undefined</code> is an explicitly-supported value of that state or property.</p>
+			<p><a>States</a> and <a>properties</a> specifically required for the <a>role</a> and subclass roles. Content authors MUST provide a non-empty <span>value</span> for required states and properties, except if current value is the default for that state or property for the role. Content authors MUST NOT use the value <code>undefined</code> for required states and properties, unless <code>undefined</code> is an explicitly-supported value of that state or property.</p>
 			<p>When an <a>object</a> inherits from multiple ancestors and one ancestor indicates that property is supported while another ancestor indicates that it is required, the property is required in the inheriting object.</p>
 			<p class="note">A host language attribute with the appropriate <a href="#implicit_semantics">implicit WAI-ARIA semantic</a> fulfills this requirement.</p>
 		</section>
 		<section id="supportedState">
 			<h3>Supported States and Properties</h3>
-			<p><a>States</a> and <a>properties</a> specifically applicable to the <a>role</a> and child roles. <a>User agents</a> MUST map all supported states and properties for the role to an accessibility API. Content authors MAY provide <span>values</span> for supported states and properties, but need not in some cases where default values are sufficient.</p>
+			<p><a>States</a> and <a>properties</a> specifically applicable to the <a>role</a> and child roles. <a>User agents</a> MUST map all supported states and properties for the role to an accessibility API. If the state or property is undefined, and it has a default value for the role, <a>User agents</a> MUST use the default value. Content authors MAY provide <span>values</span> for supported states and properties, but need not in some cases where default values are sufficient.</p>
```

Comment:
This sentence needs combining with the subsequent sentence. I'm not sure how to read the 2 together.

Reply:
Perhaps the last sentence belongs after the definition clause? i.e.
```
<p><a>States</a> and <a>properties</a> specifically applicable to the <a>role</a> and child roles. Content authors MAY provide <span>values</span> for supported states and properties, but need not in some cases where default values are sufficient. <a>User agents</a> MUST map all supported states and properties for the role to an accessibility API. If the state or property is undefined, and it has a default value for the role, <a>User agents</a> MUST use the default value.</p>
```

Reply:
Easier to read like this:
> States and properties specifically applicable to the role and child roles. Content authors MAY provide values for supported states and properties, but need not in some cases where default values are sufficient. User agents MUST map all supported states and properties for the role to an accessibility API. If the state or property is undefined, and it has a default value for the role, User agents MUST use the default value.

Reply:
Should the second instance of "user agents", the one after "role [comma]", be lowercase? That aside, this looks good to me too.

```suggestion
			<p><a>States</a> and <a>properties</a> specifically applicable to the <a>role</a> and child roles. <a>User agents</a> MUST map all supported states and properties for the role to an accessibility API. If the state or property is undefined, and it has a default value for the role, <a>user agents</a> MUST use the default value. Content authors MAY provide <span>values</span> for supported states and properties, but need not in some cases where default values are sufficient.</p>
```